### ISSUE TITLE
reboot server

### ISSUE THREAD

**Issue by avilella** [veidenberg/pline#3]
Hi,

Another small one that may be very obvious to a more experience javascript user:

Every time I change something in the code, I interrupt the running server, e.g. with Ctrl+c, then restart it. By default, it opens up a Firefox window, which I rather not have opening up (I do a `sleep 2 && pkill firefox`).

What's the recommended way of rebooting/restarting the server during code development? Is there a way for Firefox not to kick in after every reboot?

Also, is there a command-line to restart a second server on the same machine with a different port for the development server, while the "production" server keeps on for the users?

Thanks in advance

**Comment by veidenberg** [comment]
The behavior of the Pline server is defined in `server_settings.cfg` (options described in the file comments).
Set the `openbrowser` and `port` options in the configfile and restart the server or pass the options as parameters (see `python_server.py --help`).
I should to add the server documentation to the Pline website :)

**Comment by avilella** [comment]
If I change the json definition in one of the tools inside the plugins/ folder, while the server is running... Does the server pick up the changes automatically? If so, does it happen every time someone clicks refresh or F5 on the client browser window? Thanks for clarifying.

**Comment by veidenberg** [comment]
Yes, that's correct. If you refresh the browser window, the webapp setup communication (browser<=>Pline<=>JSON) updates all the available plugins.
